react/features/filmstrip/components/web/StatusBar.js

```diff
@@ -0,0 +1,112 @@
+import { getCurrentLayout, LAYOUTS } from '../../../video-layout';
+
+import AudioMutedIndicator from './AudioMutedIndicator';
+import ModeratorIndicator from './ModeratorIndicator';
+import VideoMutedIndicator from './VideoMutedIndicator';
+
+declare var interfaceConfig: Object;
+
+/**
+ * The type of the React {@code Component} props of {@link StatusBar}.
+ */
+type Props = {
+
+    /**
+     * The current layout of the filmstrip.
+     */
+    _currentLayout: string,
+
+    /**
+     * Indicates if the moderator indicator should be visible or not.
+     */
+    _showModeratorIndicator: Boolean,
+
+    /**
+     * Indicates if the audio muted indicator should be visible or not.
+     */
+    showAudioMutedIndicator: Boolean,
+
+    /**
+     * Indicates if the video muted indicator should be visible or not.
+     */
+    showVideoMutedIndicator: Boolean,
+
+    /**
+     * The ID of the participant for which the status bar is rendered.
+     */
+    participantID: String
+};
+
+/**
+ * React {@code Component} for showing the status bar in a thumbnail.
+ *
+ * @extends Component
+ */
+class StatusBar extends Component<Props> {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const {
+            _currentLayout,
+            _showModeratorIndicator,
+            showAudioMutedIndicator,
+            showVideoMutedIndicator
+        } = this.props;
+        let tooltipPosition;
+
+        switch (_currentLayout) {
+        case LAYOUTS.TILE_VIEW:
+            tooltipPosition = 'right';
+            break;
+        case LAYOUTS.VERTICAL_FILMSTRIP_VIEW:
+            tooltipPosition = 'left';
+            break;
+        default:
+            tooltipPosition = 'top';
+        }
+
+        return (
+            <div>
+                { showAudioMutedIndicator ? <AudioMutedIndicator tooltipPosition = { tooltipPosition } /> : null }
+                { showVideoMutedIndicator ? <VideoMutedIndicator tooltipPosition = { tooltipPosition } /> : null }
+                { _showModeratorIndicator ? <ModeratorIndicator tooltipPosition = { tooltipPosition } /> : null }
+            </div>
+        );
+    }
+}
+
+/**
+ * Maps (parts of) the Redux state to the associated {@code StatusBar}'s props.
+ *
+ * @param {Object} state - The Redux state.
+ * @param {Object} ownProps - The own props of the component.
+ * @private
+ * @returns {{
+ *     _currentLayout: string,
+ *     _showModeratorIndicator: boolean
+ * }}
+*/
+function _mapStateToProps(state, ownProps) {
+    const { participantID } = ownProps;
+
+    // Only the local participant won't have id for the time when the conference is not yet joined.
+    const participant = participantID ? getParticipantById(state, participantID) : getLocalParticipant(state);
+
+    return {
+        _currentLayout: getCurrentLayout(state),
+        _showModeratorIndicator:
```

Comment:
On mobile we don't show the indicator if everyone is a moderator: https://github.com/jitsi/jitsi-meet/blob/15fb8a152520298580f69fdfa3579dffce2834e7/react/features/filmstrip/components/native/Thumbnail.js#L261

Reply:
I've preserved the current behaviour on web which is:
We show the moderator icon unless the interface_config's `DISABLE_FOCUS_INDICATOR` is true.

I guess it will be good if we have consistency across web and mobile but this can be done trough another PR since this is not related to the purpose of this one!

Reply:
Why not do it now that we are here? It takes 1 minute to ask and another one to implement. 🤷‍♂

Reply:
IMHO the current implementation is a little bit more flexible. Also I'd still prefer another PR.